### DISCUSSION TITLE
TimedStream (how to)

### DISCUSSION THREAD

**Discussion by vintagecardriver** [pschatzmann/arduino-audio-tools#1678] [Q&A] [answered]
Hi Phil,
as I wrote in that question ( https://github.com/pschatzmann/arduino-audio-tools/discussions/1619 ), I want to jump to a defined moment (for example to Second 52) of the song, when playing mp3-file from sd-card.
You gave me the hint for TimedStream, but I can't find any example how to use the timedStream call.
Would you be so kind, please, and give me a short example using timedStream?
It would be great!
Thank you very much and best regards

**Comment by pschatzmann** [comment, marked as answer]
Just look at the class documentation: there are several way to achieve this. Either define the start time in the constructor or by calling the corresponding methods.

Just wrap the output class with the TimedStream, define the start and send the output to the TimedStream...
